Tests/TuistKitTests/Services/Cache/CachePrintHashesServiceTests.swift

```diff
@@ -84,15 +96,28 @@ final class CachePrintHashesServiceTests: TuistUnitTestCase {
 
     func test_run_gives_correct_artifact_type_to_hasher() throws {
         // When
-        _ = try subject.run(path: path, xcframeworks: true)
+        _ = try subject.run(path: path, xcframeworks: true, profile: nil)
 
         // Then
         XCTAssertEqual(graphContentHasher.invokedContentHashesParameters?.cacheOutputType, .xcframework)
 
         // When
-        _ = try subject.run(path: path, xcframeworks: false)
+        _ = try subject.run(path: path, xcframeworks: false, profile: nil)
 
         // Then
         XCTAssertEqual(graphContentHasher.invokedContentHashesParameters?.cacheOutputType, .framework)
     }
+
+    func test_run_gives_correct_cache_profile_type_to_hasher() throws {
+        // When
+        let profile: Cache.Profile = .test(name: "Simulator", configuration: "Debug")
```

Comment:
nitpick: This should probably be in `Given` section and `When` should be used before `subject.run(...)`